kujira/app/services/enevt-handler-service.js

```diff
@@ -0,0 +1,33 @@
+
+import Ember from 'ember';
+var run= Ember.run;
+var socket;
+var hostName ='';
+
+export default Ember.Service.extend(Ember.Evented,{
+
+  onInit: function(){
+
+    socket = io.connect(hostName);
+    socket.on('message', this.onMessage, this);
+    socket.on('close', function(event) {
+       socket= io.connect(hostName);
+   }, this);
+
+  }.on(init),
+  onMessage: function(message){
+
+    var obj= JSON.parse(message);
+
+    var type = Ember.compare(message.eventType, 'osdAdded');
+    {{#if type}}
```

Comment:
Why do you use handlebars syntax in js? -> Use standart if (condition) {} else {}